pkg/reconciler/v1alpha1/service/service_test.go

```diff
@@ -357,6 +378,35 @@ func TestReconcile(t *testing.T) {
 		WantEvents: []string{
 			Eventf(corev1.EventTypeNormal, "Updated", "Updated Service %q", "all-ready"),
 		},
+	}, {
+		Name: "runLatest - route and config ready, propagate ready-2",
```

Comment:
We should just make these changes to the test above this vs. have 2.  This new test is more accurate than the old one.

Reply:
I guess. Removed one of them.